src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -184,6 +189,29 @@ private String commandStringGenerator(
     return commandString.toString();
   }
 
+  /**
+   * Reads Serverconfig from File pathToResourceDocker/ssh.rc
+   */
+  private void readServerConfig() {
+    String configFilePath = pathToResourceDocker + "/ssh.rc";
+    try {
+      List<String> lines = Files.readAllLines(Paths.get(configFilePath), Charset.forName("UTF-8"));
+      for (String line : lines) {
+        if (line.contains("p2p_server=") && !line.endsWith("=")) {
```

Comment:
Okay, sounds good!
Since I have to prepare for the presentation, maybe François can look at the logger or we implement it after assessment :)